handsome-hedgehogs/workforce-management
# Optimize

## Status
Choose:
**READY**

## Description
fixed some minor bugs and inconsistency in data manipulation

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.
1. git fetch
2. git checkout optimize
3. run program
4. confirm that decommissioning a computer, creates and enddate for all employees using that computer
5. confirm minimum capacity for training programs being edited or created is 1
6. confirm all employee, trainingprogram, and computer date type data is entered using a date helper box
